CONTRIBUTING.md

```diff
@@ -237,18 +237,18 @@ $ docker-compose up --build -d
 ```
 
 If you're working on a dependency that doesn't live under this repository (for instance,
-`baggageclaim`), you'll need to update `go.mod` with a `replace` directive with the exact
+`dex`), you'll need to update `go.mod` with a `replace` directive with the exact
 reference that the module lives at:
 
 ```sh
-# after pushing to the `sample` branch in your baggageclaim fork,
+# after pushing to the `sample` branch in your dex fork,
 # try to fetch the module revision and get the version.
-$ go mod download -json github.com/your-user/baggageclaim@sample | jq '.Version'
-go: finding github.com/cirocosta/baggageclaim sample
+$ go mod download -json github.com/your-user/dex@sample | jq '.Version'
+go: finding github.com/cirocosta/dex sample
 "v1.3.6-0.20190315100745-09d349f19891"
 
 # with that version, update `go.mod` including a replace directive
-$ echo 'replace github.com/concourse/baggageclaim => github.com/your-user/baggageclaim v1.3.6-0.20190315100745-09d349f19891' \
+$ echo 'replace github.com/concourse/dex => github.com/your-user/dex v1.3.6-0.20190315100745-09d349f19891' \
```

Comment:
`v1.3.6-0.20190315100745-09d349f19891` here is a version of baggageclaim right? Could we make it a version of Dex?